docs/deployment/README.md

```diff
@@ -42,77 +42,86 @@ The process involves:
+To use the specific version of the package installed during deployment, point the `source` key in the lambda config to `node_modules/@cumulus/<package-name>/dist`. This location may vary between packages, so consult the README in each. For example:
 
-If you're trying to work with a certain version of a cumulus package or task, the version can be specified in `package.json` under dependencies. We use semantic versioning (major/minor/patch). You can also configure for automatic updates. Use `^` to update minor/patch versions automatically and `~` to automatically update patch versions. For example:
+    SyncGranule:
+      source: 'node_modules/@cumulus/sync-granule/dist/'
 
-    "@cumulus/sync-granule": "^1.0.0"
+##### Option 2: Make local copy of the `Cumulus` Repo and prepare it.
 
-Then run:
+Clone repository
 
-    $ npm install
+    $ git clone https://github.com/nasa/cumulus.git
 
-To add a new package, install via npm. Without a version specified, it will automatically install the latest version. For example:
+Change directory to the repository root
 
-    $ npm install @cumulus/deployment
+    $ cd cumulus
 
-**Note**: The npm install command will add the [kes](http://devseed.com/kes/) utility to the `<daac>-deploy`'s `node_packages` directory and will be utilized later for most of the AWS deployment commands
+Optionally, If you are deploying a particular version(tag), ref or branch of Cumulus core, you should check out that particular reference
 
-To use the specific version of the package installed during deployment, point the `source` key in the lambda config to `node_modules/@cumulus/<package-name>/dist`. This location may vary between packages, so consult the README in each. For example:
+    $ git checkout \<ref/branch/tag\>
 
-    SyncGranule:
-      source: 'node_modules/@cumulus/sync-granule/dist/'
+Install and configure the local build environment and dependencies using npm
+
+    $ nvm use
+    $ npm install
+    $ npm run ybootstrap
 
-The [`Cumulus`](https://github.com/nasa/cumulus) project contains default configuration values in `cumulus/packages/deployment/app.example`, however these need to be customized for your Cumulus app.
+Build the Cumulus application
+
+    $ npm run build
+
+To run the Cumulus deployment with the local code instead of the npm package, use `npm link` 
+
+    $ cd ~/<daac-deployment-directory>
+    $ npm link ../cumulus/packages/deployment/ @cumulus/deployment
 
 ##### Copy the sample template into your repository {#copy-template}
 
 Begin by copying the template directory to your project. You will modify it for your DAAC's specific needs later.
 
-    $ cp -r ../cumulus/packages/deployment/app.example ./app
+    $ cp -r ./node_modules/@cumulus/deployment/app.example ./app
+
+The [`Cumulus`](https://github.com/nasa/cumulus) project contains default configuration values in the `app.example` folder, however these need to be customized for your Cumulus app.
```

Comment:
I would put this sentence before the "Begin by copying" since this tells you why we're copying.